x/capability/types/keys.go

```diff
@@ -14,7 +14,7 @@ const (
 	StoreKey = ModuleName
 
 	// MemStoreKey defines the in-memory store key
-	MemStoreKey = "mem_capability"
+	MemStoreKey = "memory:capability"
```

Comment:
This is only used for testing, we might want to move it over. Same goes for StoreKey.

Reply:
Why was this change necessary, just curious?

Reply:
Because now the store keys are being registered through this function that has a fixed format: https://github.com/cosmos/cosmos-sdk/blob/8eaff8fadd408762fa5a3c5a40969bda4548057e/runtime/module.go#L115-L119

I think this key is now deprecated and used only in tests because we don't use it anywhere else.

Reply:
got it. If so, can we prefix the variable with `Test` or something? Or maybe just remove it completely and define it directly in tests?

Reply:
I think we should move it over to test. We still need it to create the MemoryStoreKey pass it to the Keeper in tests. @aaronc this is what I pinged you about on Slack

Reply:
So this is sort of still needed for people using the traditional app.go setup. We should try not to break that because we're not trying to force everyone to change all at once. We should maybe even have a copy of simapp/app.go before the app wiring (maybe simapp/legacy) that checks to make sure that way still works

Reply:
> We should maybe even have a copy of simapp/app.go before the app wiring (maybe simapp/legacy) that checks to make sure that way still works

I like that idea!

Reply:
Ok, just opened #12068

Reply:
I think it's okay to change the value of `MemStoreKey` just not to delete the const